src/meteodatalab/operators/regrid.py

```diff
@@ -485,11 +486,13 @@ def iconremap(
 
     gx, gy = np.meshgrid(dst.x, dst.y)
     transformer_dst = Transformer.from_crs(dst.crs.wkt, utm_crs)
-    points_dst = transformer_dst.transform(gx.flat, gy.flat)
+    points_dst = transformer_dst.transform(gy.flat, gx.flat)
```

Comment:
Which coordinate system are you using for your test?

Reply:
geolatlon

Reply:
okay, I would rather add `always_xy` to the previous line because this change is breaking support for all CRSs that are in the traditional order